### PR TITLE
Create wtf.html

### PR DESCRIPTION
**Pull request by techniboy**

### CODE REVIEW

File: wtf.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<html>
+<header><title>wtf</title></header>
```
Comment (kaada) [inline wtf.html:2]: Nice                  spacing   and   code              style

bitmoji

Comment (techniboy) [reply wtf.html:2]: thanks dud!

Comment (kaada) [reply wtf.html:2]: Wow wtf is that comment of mine, that's so harsh

File: wtf.html [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+<body>
+  wtf bro?
```
Comment (kaada) [inline wtf.html:4]: 👀

**Pull request closed without merging**